data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+
+
+mission "Navy vs Pirate"
+	landing
+	name "Destroy <npc>"
+	description "A pirate vessel named <npc> has broken through the Navy defenses in Alnitak and is currently in the Saiph system. Destroy it before it can move further into Republic space and return to <planet> for additional orders."
+	to offer
+		has "Navy Pirate Patrol 3: done"
+		not "chosen sides"
+	source "Farpoint"
+	waypoint "Saiph"
+	on fail
+		dialog `You have failed an essential Navy mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+
+	npc kill
+		personality heroic staying nemesis target unconstrained opportunistic appeasing mining harvests
+		government "Pirate"
+		system "Saiph"
+		names "pirate"
+		fleet
+			variant
+				"Leviathan"
+			variant
+				"Leviathan (Heavy)"
+			variant
+				"Leviathan (Laser)"
+
+		dialog "The <npc> has been eliminated. Return to <destination> for additional orders."
+	on offer
+		conversation
+			`Farpoint feels a lot different when the entire base is on alert. Normally, you could easily see from one side of the compound to the other and only have the odd soldier or vehicle break your vision. Since the pirate ship that broke through, however, the farthest you can see is maybe 50 feet due to the mass of guards, crewmen, machinery and munitions.`
+			`	Eventually, you find the Admiral near his ship, the N.S. Peacemaker; other crewmen are busily loading his ship while he stares intently at a clipboard, clearly distraught about something.`
+			choice
+				`	(Ask him what he's looking at.)`
+					goto ask
+				`	(Wait for him to finish reading.)`
+					goto wait
+
+			label ask
+			apply	
+				set "ranulph's background"
+			`	"Something wrong, Admiral?" you inquire, concerned.`
+			`	"Oh, it's Hines," he dismissively comments as he looks up from his clipboard. "I'm just reading some performance commentary from his peers."`
+			`	Your face twists into another uneasy expression. "I'm guessing that falls under the category of 'not good.'"`
+			`	"Ranulph Hines always been a handful," the admiral replies, setting the clipboard down. "The man seems to have it in his head that by climbing the ranks quickly, he'll be seen as some sort of hero. While I admire his efforts to do his best, his means of getting there have been..." Danforth makes an aside glance, as if checking to see if Ranulph is in earshot. "Well, to say that it's underhanded would be a slight understatement."`
+			choice
+				`	"If you don't mind me asking, Admiral, exactly how bad are we talking?"`
+					goto ranulph
+				`	"I see."`
+					goto pirateHunt
+
+			label ranulph
+			apply
+				set "excessive force"
+			`Danforth sighs sadly as he shakes his head. "I'll be honest with you, <first> - it wasn't as bad when he first joined, but recently, some of the things Ranulph's done are borderline terroristic. Still fully legal within the bounds of Galactic Law, but extremely questionable from a moral standpoint."`
```

Comment:
I might've missed something earlier, but discussing this sort of thing seems like gross misconduct on Danforth's part unless you've established a close, trusting relationship. Even then it should probably come with a verbalised disclaimer.